vyper/parser/lll_node.py

```diff
@@ -91,9 +91,15 @@ def __init__(self,
                 # at pop time; this makes `break` easier to handle
                 self.gas = gas + 2 * (outs - ins)
                 for arg in self.args:
-                    # TODO: remove dead code.
-                    # if arg.valency == 0:
-                    #     raise Exception("Can't have a zerovalent argument to an opcode or a pseudo-opcode! %r: %r" % (arg.value, arg))  # noqa: E501
+                    # pop and pass are used to push/pop values on the stack to be
+                    # consumed for private functions, therefore we whitelist this as a zero valency
+                    # allowed argument.
+                    zero_valency_whitelist = {'pass', 'pop'}
+                    if arg.valency == 0 and arg.value not in zero_valency_whitelist:
+                        raise Exception(
+                            "Can't have a zerovalent argument to an "
```

Comment:
Please, please, please a better error message here lol

Reply:
This is an internal API, I don't think we can make it any more clear? Any suggestions?

Reply:
I think we can just add, 'please file a bug report' ?

Reply:
@fubuloubu Added: file a bug report.